scripts/fobnail-mount.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description="Mount Fobnail-protected disk"
+After=network.target
+StopWhenUnneeded=yes
+
+[Service]
+EnvironmentFile=/etc/fobnail.cfg
+RuntimeDirectory=fobnail
+RuntimeDirectoryMode=0600
+Type=simple
+RemainAfterExit=yes
+ExecStart=/usr/share/fobnail/mount.sh
+ExecStop=/usr/share/fobnail/umount.sh
```

Comment:
How can this service stopped? Just by manually calling `systemctl stop`?

Reply:
Yes, it can be stopped with `systemctl stop`, also it is stopped automatically after user unplugs Fobnail.